shapeshift/web
# feat: basic THORChain support

## Description

Add basic thorchain support (send, receive, balance, transaction history)

## Notice




## Pull Request Type


## Issue (if applicable)

closes https://github.com/shapeshift/web/issues/2663

## Risk

Low - additive feature

## Testing

- Navigate to RUNE asset and generate a new receive address
- Request funding from Ops
- Observe incoming transaction in tx history
- Validate balance
- Create a send transaction to another address
- Observe outgoing transaction in tx history
- Validate balance

Known Issues:
- Balance charts will be off with RUNE enabled for any of the following reasons:
  - Swap transaction are not currently being picked up
  - Any transactions made before the `thorchain-mainnet-v1` upgrade will not show
  - Fee showing as 0.02 RUNE instead of 0.04 RUNE

### Engineering

See above

### Operations

See above and regression test cosmos and osmosis on all supported wallets

## Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/35275952/188751216-c82d7226-4b8a-40dc-a73e-2728ceace98e.png)
![image](https://user-images.githubusercontent.com/35275952/188751289-46c67f5a-ebd4-4031-82d5-7e4f7f6c9813.png)
![image](https://user-images.githubusercontent.com/35275952/188755114-f254dd06-0bad-4d8f-b040-45d2d0d11604.png)
![image](https://user-images.githubusercontent.com/35275952/188755164-59d52611-71c7-40d8-8c36-0206e08270c3.png)
![image](https://user-images.githubusercontent.com/35275952/188755237-1ba38472-918c-47b7-9648-7d2acf8d21bb.png)
![image](https://user-images.githubusercontent.com/35275952/188917614-f8fa5e80-46f0-4cf5-9633-3cf2987986ef.png)
![image](https://user-images.githubusercontent.com/35275952/188918271-e641c4e1-5476-4631-b160-8d181645957c.png)
![image](https://user-images.githubusercontent.com/35275952/188984907-2a439d2c-542e-4ae2-b62b-5082e1fc2aec.png)
